packaging/CPackOptions.cmake

```diff
@@ -0,0 +1,65 @@
+# Copyright 2019-2019 the openage authors. See copying.md for legal info.
+
+# Keeps all required CPack configuration for the Generators in one place and
+# CPack calls this file for every generator, keep this in mind, if you use
+# variables or functions outside of the conditional branch
+
+
+########################################################################
+# Continuous Integration
+
+# Filename addition for architecture, set from environment variables
+set(CPACK_PACKAGE_ARCHITECTURE "-$ENV{TARGET_PLATFORM}")
+
+# Filename addition for nightly-builds, set from environment variables
+if(DEFINED ENV{IS_NIGHTLY})
+    set(CPACK_PACKAGE_VERSION "${VERSION_FULL_STRING}")
```

Comment:
for nightly builds we set the full version string for easier identification of new versions